src/App.tsx

```diff
@@ -1,24 +1,162 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { UserWarning } from './UserWarning';
+import { TodoFooter } from './components/TodoFooter/TodoFooter';
+import { TodoHeader } from './components/TodoHeader/TodoHeader';
+import { TodoList } from './components/TodoList/TodoList';
+import * as todoService from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoError } from './components/TodoError/TodoError';
+import { Errors } from './types/Errors';
+import { Filter } from './types/Filter';
+import { prepareTodos } from './utils/helpers';
 
-const USER_ID = 0;
+const USER_ID = 12054;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterStatus, setFilterStatus] = useState<Filter>(Filter.All);
+  const [errorType, setErrorType] = useState<Errors | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [processingTodoIds, setProcessingTodoIds] = useState<number[]>([]);
+  const [todoTitle, setTodoTitle] = useState('');
+
+  useEffect(() => {
+    todoService.getTodos(USER_ID)
+      .then(response => {
+        setTodos(response);
+      })
+      .catch((error) => {
+        throw error;
```

Comment:
Instead of throwing an error, set an appropriate message to `errorType` state to show the user what's wrong